<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertificateView</class>
 <widget class="QDialog" name="CertificateView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SSL certificate</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QGridLayout" columnstretch="0,0,0,0,0,0,0">
       <item row="8" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Organization:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>SHA1 fingerprint:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="4" colspan="3">
        <widget class="QLabel" name="iou_label">
         <property name="text">
          <string notr="true">IOU</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="7">
        <widget class="QLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Issued by</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Expires on:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="4" colspan="3">
        <widget class="QLabel" name="orgunit_label">
         <property name="text">
          <string notr="true">OU</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="3">
        <widget class="QLabel" name="org_label">
         <property name="text">
          <string notr="true">O</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="4" colspan="3">
        <widget class="QLabel" name="io_label">
         <property name="text">
          <string notr="true">IO</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="3">
        <widget class="QLabel" name="cn_label">
         <property name="text">
          <string notr="true">CN</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="15" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>MD5 fingerprint:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Organization:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Organization unit:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Issued on:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Organization unit:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Common name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Serial number:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="4" colspan="3">
        <widget class="QLabel" name="md5_label">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string notr="true">00000000000000000000000000000000</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="7">
        <widget class="QLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Issued to</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4" colspan="3">
        <widget class="QLabel" name="icn_label">
         <property name="text">
          <string notr="true">ICN</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLabel">
         <property name="text">
          <string>Common name:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="4" colspan="3">
        <widget class="QLabel" name="sha1_label">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string notr="true">0000000000000000000000000000000000000000</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="4" column="4" colspan="3">
        <widget class="QLabel" name="sn_label">
         <property name="text">
          <string notr="true">SN</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="7">
        <widget class="QLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Validity period</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" rowspan="2" colspan="7">
        <widget class="QLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fingerprints</string>
         </property>
        </widget>
       </item>
       <item row="11" column="4" colspan="3">
        <widget class="QLabel" name="issuedon_label">
         <property name="text">
          <string notr="true">0000</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="4" colspan="3">
        <widget class="QLabel" name="expireson_label">
         <property name="text">
          <string notr="true">0000</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTextEdit" name="certificateText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CertificateView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CertificateView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
